acra-censor/acra-censor_implementation.go

```diff
@@ -29,21 +29,25 @@ func (acraCensor *AcraCensor) ReleaseAll() {
 }
 
 func (acraCensor *AcraCensor) HandleQuery(query string) error {
+	queryWithHiddenValues, err := handlers.RedactSQLQuery(query)
+	if err == handlers.ErrQuerySyntaxError && acraCensor.ignoreParseError {
+		log.WithError(err).Infof("parsing error on query (first %v symbols): %s", handlers.LogQueryLength, handlers.TrimStringToN(queryWithHiddenValues, handlers.LogQueryLength))
```

Comment:
1. Start logs with capital letter

2. I would change log to "Got error while parsing query ..."